.gitlab/binary_build/cluster_agent_cloudfoundry.yml

```diff
@@ -1,7 +1,5 @@
 ---
 cluster_agent_cloudfoundry-build_amd64:
-  rules:
-    !reference [.on_a7]
```

Comment:
If you remove this rule these jobs will be created in the MQ, no?

Reply:
Good catch, the [test](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/pipelines/31117488) was also relevant. Update on-going, thanks a lot